packages/core/src/data-editor/data-editor.tsx

```diff
@@ -3321,7 +3321,10 @@ const DataEditorImpl: React.ForwardRefRenderFunction<DataEditorRef, DataEditorPr
             let cancelled = false;
             if (onKeyDownIn !== undefined) {
                 onKeyDownIn({
-                    ...event,
+                    ...event, // FIXME: Mutate
```

Comment:
Can the `FIXME: Mutate` comment be removed?

Reply:
done!